update.go

```diff
@@ -215,7 +215,14 @@ func loadNewCLI() {
 	current, err := os.Stat(BinPath)
 	must(err)
 	if !os.SameFile(current, expected) {
-		execBin(expected.Name(), Args...)
+		bin := expectedBinPath()
+		Debugf("Executing %s\n", bin)
+		cmd := exec.Command(bin, Args[1:]...)
```

Comment:
it's a little cleaner to exec over rather than load up a second process, but the old code was actually written backwards I think because syscall.exec doesn't even work on windows I don't think